### ISSUE TITLE
Ability to change all prices fast

### ISSUE THREAD

**Issue by ZvezdaDark** [CorsixTH/CorsixTH#2316]
### What feature or enhancement would you like to add?

Add buttons to increase / decrease in list of prices (I forgot how its called, like place where you can increase prices of cures). Game has it already in cheat menu, but we need this feature somewhere else.

### How would this feature benefit the gameplay of CorsixTH?

It will be easy and fast now to change all prices 

### Was the feature in the original game?

Yes

### What version of CorsixTH are you using?

0.66

### Does this feature/enhancement relate to a current problem?

No

**Comment by lewri** [comment]
Yes it's not a cheat per se I agree. More an ease of use function.

**Comment by lukejames1111** [comment]
I agree, this would definitely make things easier, but I don't recall this being in the original game 😂

**Comment by MandrakeHorse** [comment]
+1. Sometimes I'll want to raise the prices of everything in the Drug Casebook to 105-110% to make more money and it's very time-consuming to adjust every treatment individually. The Casebook UI doesn't have room to add more buttons, but perhaps Shift+clicking the plus/minus signs could increase/decease all prices simultaneously by 1%?

**Comment by Matroftt** [comment]
It could be implemented through charts menu, like that
![image](https://github.com/CorsixTH/CorsixTH/assets/91145555/d5eee71d-362b-4b58-83e1-bacc7d10f4a4)
Although the player most likely wont notice it
